prow/plugins/help/help.go

```diff
@@ -30,19 +30,32 @@ import (
 const pluginName = "help"
 
 var (
-	helpLabel         = "help wanted"
-	helpRe            = regexp.MustCompile(`(?mi)^/help\s*$`)
-	helpRemoveRe      = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
-	helpGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
-	helpMsgPruneMatch = "This request has been marked as needing help from a contributor."
-	helpMsg           = `
+	helpLabel                  = "help wanted"
+	goodFirstIssueLabel        = "good first issue"
+	helpRe                     = regexp.MustCompile(`(?mi)^/help\s*$`)
+	helpRemoveRe               = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
+	helpGoodFirstIssueRe       = regexp.MustCompile(`(?mi)^/help good-first-issue\s*$`)
+	helpGoodFirstIssueRemoveRe = regexp.MustCompile(`(?mi)^/help remove-good-first-issue\s*$`)
+	helpGuidelinesURL          = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
+	helpMsgPruneMatch          = "This request has been marked as needing help from a contributor."
+	helpMsg                    = `
 This request has been marked as needing help from a contributor.
 
 Please ensure the request meets the requirements listed [here](` + helpGuidelinesURL + `).
 
 If this request no longer meets these requirements, the label can be removed
 by commenting with the ` + "`/remove-help`" + ` command.
 `
+	goodFirstIssueGuidelinesURL = "TODO"
```

Comment:
Will add this after @carolynvs updates the docs.

Reply:
I actually kind of expect this to be in the same doc as help wanted

Reply:
> I actually kind of expect this to be in the same doc as help wanted

updated.